src/main/java/com/google/step/servlets/CommentServlet.java

```diff
@@ -0,0 +1,79 @@
+package com.google.step.servlets;
+
+import com.google.step.datamanager.CommentManager;
+import com.google.step.datamanager.CommentManagerDatastore;
+import com.google.step.model.Comment;
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that handles individual comments. */
+@WebServlet("/api/comments/*")
+public class CommentServlet extends HttpServlet {
+
+  private CommentManager manager = new CommentManagerDatastore();
+
+  public CommentServlet(CommentManager commentManager) {
+    manager = commentManager;
+  }
+
+  /** Deletes the comment with the given id parameter */
+  @Override
+  public void doDelete(HttpServletRequest request, HttpServletResponse response)
+      throws IOException {
+    long id;
+    try {
+      id = Long.parseLong(request.getPathInfo().substring(1));
```

Comment:
might need to check if `request.getPathInfo()` is null, for when user goes to `/api/comments` directly without anything after /*

Reply:
theres another servlet for that url (getpostservlet) i transferred the doget over